tests/execution/graphql-conformance-http-015.md

```diff
@@ -0,0 +1,182 @@
+# Optional input fields
+
+```graphql @config
+schema
+  @server(port: 8001, queryValidation: false, hostname: "0.0.0.0")
+  @upstream(baseURL: "http://upstream/", httpCache: 42) {
+  query: Query
+}
+
+type Query {
+  user(id: ID!): User! @http(path: "/user", query: [{key: "id", value: "{{.args.id}}"}])
+}
+
+type User {
+  id: ID!
+  name: String!
+  profilePic(size: Int! = 100, width: Int, height: Int = 100): String!
+    @expr(body: "{{.value.id}}_{{.args.size}}_{{.args.width}}_{{.args.height}}")
+  featuredVideo(video: VideoSize! = {width: 1600, height: 900}): String!
+    @expr(body: "video_{{.value.id}}_{{.args.video.width}}_{{.args.video.height}}_{{.args.video.hdr}}")
+  featuredVideoPreview(video: VideoSize!): String!
+    @expr(body: "video_{{.value.id}}_{{.args.video.width}}_{{.args.video.height}}_{{.args.video.hdr}}")
+  searchComments(query: [[String!]!]! = [["today"]]): String! @expr(body: "video_{{.value.id}}_{{.args.query}}")
+}
+
+input VideoSize {
+  width: Int!
+  height: Int!
+  hdr: Boolean
```

Comment:
this test differs from its companion test for graphql because there is no default values inside input type

Reply:
good catch :+1: